chapters/en/slides/module5/module5_05.Rmd

```diff
@@ -0,0 +1,393 @@
+```
+
+type: slides
+
+
+# Effective use of color for quantitative data
+
+Notes:
+This slide deck describes how to choose color to accurately represent numerical data.
+
+---
+
+## Repetition of hue, lightness, and saturation
+
+<center><img src="/module5/hsl-questions.png" width="50%"></img></center>
+
+Notes:
+In the last slide deck we varied hues to represent categorical values,
+with a focus on the distinction between color hues/categories.
+When choosing a color scheme to represent quantitative data,
+we will instead focus on sequential, perceptually uniform changes in lightness.
+
+Sequential changes in lightness ensures that uniform changes in the values of the data
+are perceived as uniform changes in the color space.
+In other words,
+we want it to be intuitive 
+how the colors relate the numerical values they represent,
+so that we can accurately understand how the data values vary
+when looking at the changing colors.
+
+---
+
+## Lightness is more effective than hue to communicate changes in quantitative data
+
+```{python}
+# TODO hide code
+import altair as alt
+from vega_datasets import data
+
+cars = data.cars()
+chart = alt.Chart(cars).mark_point(size=50, filled=True).encode(
+    x='Horsepower',
+    y='Miles_per_Gallon',
+    color=alt.Color('Origin', title=None))
+((chart.encode(alt.Color('Horsepower', scale=alt.Scale(scheme='tableau10')))
+| chart.encode(alt.Color('Horsepower')))
+ .resolve_scale(color='independent'))
+```
+
+Notes:
+In the plots on this slide,
+we have encoded the same variable (Horsepower) to be represented by both the x-axis and the color.
+This is rarely done in practice,
+but it is useful to contrast and understand color schemes
+because we know that the colors of the points should change in a perceptually uniform manner
+as we move from left to right along the x-axis.
+
+In the leftmost plot in this slide we used the default categorical color scheme.
+As you can see this is not at all well suited to represent numerical data
+as the sharp transitions between hues makes it appear as if there are groups in the data,
+whereas we know that these values are changing gradually from left to right on the x-axis.
+
+In the plot to the right,
+we have used the default sequential color scheme for quantitative data in Altair.
+This conveys the gradual change in the Horsepower values in a perceptually uniform manner.
+We can clearly see that color is changing gradually from light to dark along the x-axis,
+although it can be hard to pinpoint exactly which number a particular color is
+(remember that color is not as good as position to convey an exact value
+even when it is easy to see the overall trend).
+
+---
+
+## Combining lightness variation with a slight variation in hue highlights detail in the data
+
+```{python}
+# TODO hide code
+((chart.encode(alt.Color('Horsepower'))
+| chart.encode(alt.Color('Horsepower', scale=alt.Scale(scheme='viridis', reverse=True))))
+ .resolve_scale(color='independent'))
```

Comment:
Please use nicely formatted axis labels, not the default column names. Units too please.

Reply:
The column names are the units here, "Miles per gallon" and "Horsepower"